src/ucs/stats/stats.c

```diff
@@ -98,6 +98,11 @@ static ucs_stats_class_t ucs_stats_root_node_class = {
     }
 };
 
+/* Statistics id to name */
+static ucs_stats_class_id_to_name_s
```

Comment:
can we allocate this array only when used by ucs_stats_class_id_to_aggregate_sum_base_offset_get?
for example, use ucs_array_t to define global auto-growing array

Reply:
@yosefe , I will update this code to use ucs_malloc() and ucs_realloc().

Reply:
Fixed.